client/src/plus/updates/login-banner.tsx

```diff
@@ -13,7 +13,7 @@ export function LoginBanner() {
       <span>
         <span>Want to use filters?</span>{" "}
         <strong>
-          Upgrade to{" "}
+          Log in to{" "}
```

Comment:
UX comment: same as above

Reply:
Adressed by changing it to "Upgrade to MDN Plus for free".